### PR TITLE
librespeed-go: Reload the daemon after modifying the tls certificate

### PR DESCRIPTION
**Pull request by muink**
Maintainer: Tianling Shen cnsztl@immortalwrt.org @1715173329 
Compile tested: x86_64, x86/64, 22.03.5
Run tested: x86_64, x86/64, 22.03.5

Description:
Make the daemon reload after the tls certificate is updated

### CODE REVIEW

File: net/librespeed-go/files/librespeed-go.init [hunk 1]
```diff
@@ -31,6 +32,10 @@ mount_jail_file() {
 	else
 		procd_add_jail_mount "$value"
 	fi
+
+	if [ "$refresh" = "1" ]; then
+		procd_append_param file "$value"
+	fi
```
Comment (1715173329) [inline net/librespeed-go/files/librespeed-go.init:38]:
```suggestion
	[ "$refresh" = "0" ] || procd_append_param file "$value"
```

btw i'd prefer `reload` over `refresh`

**Review by 1715173329 — CHANGES_REQUESTED** [review]
Please:
- Drop unnecessary description from your commit message
  `Make the daemon reload after the tls certificate is updated.` is enough
- Use a valid email address

**Review by 1715173329 — APPROVED** [review]

**Comment by BKPepe** [comment]
Requires to bump PKG_RELEASE in the Makefile.

**Comment by 1715173329** [comment]
Merged, thank you!

**Pull request merged**